app/main.ipynb

```diff
@@ -395,6 +1493,7 @@
     "        result += vector1[i] * vector2[i]\n",
     "    return result\n",
     "\n",
+    "\n",
     "vector1 = [1, 2, 3]\n",
     "vector2 = [4, 5, 6]\n",
```

Comment:
The vectors `vector1` and `vector2` are defined as lists. For consistency with NumPy operations, consider converting them to NumPy arrays before using them in the `%time np.dot(vector1, vector2)` cell.